doc/integration.md

```diff
@@ -1 +1,110 @@
-Nothing here just yet!
+#Integration Guide
+This guide includes all information necessary to receive payments from a user of your application through Klarna. In this guide, you will see how to allow the user to register his device with Klarna, change payment preferences and perform purchases.
+
```

Comment:
I think that we are missing table of contents with links (also in the IOS):
- Set API key
- Registration
- Purchase
- Preference